force-app/main/default/lwc/donationHistoryTable/donationHistoryTable.js

```diff
@@ -1,17 +1,13 @@
-import { api, LightningElement, track, wire } from 'lwc';
-import { getObjectInfo } from 'lightning/uiObjectInfoApi';
-import DATA_IMPORT from '@salesforce/schema/DataImport__c';
-import PAYMENT_FIELD from '@salesforce/schema/DataImport__c.Payment_Method__c';
-import donationHistoryDatatableAriaLabel from '@salesforce/label/c.donationHistoryDatatableAriaLabel';
-import RD2_ScheduleVisualizerColumnDate from '@salesforce/label/c.RD2_ScheduleVisualizerColumnDate';
-import getDonationHistory from '@salesforce/apex/DonationHistoryController.getDonationHistory';
-import commonAmount from '@salesforce/label/c.commonAmount';
-import donationHistoryDonorLabel from '@salesforce/label/c.donationHistoryDonorLabel';
+import { api, LightningElement, track } from "lwc";
+import donationHistoryDatatableAriaLabel from "@salesforce/label/c.donationHistoryDatatableAriaLabel";
+import RD2_ScheduleVisualizerColumnDate from "@salesforce/label/c.RD2_ScheduleVisualizerColumnDate";
```

Comment:
@bdvorachek - any qualms with reusing an RD2 specific custom label? Perhaps we should have a common label for 'Date'?

Reply:
Not super concerned about it, but I agree that we should probably have this label be `commonDate` if we're going to use it here.